### PR TITLE
fix(SerialConsole): Set console width based on columns and add auto-sizing option

### PR DESCRIPTION
**Pull request by jschuler**
Addresses issue https://github.com/patternfly/patternfly-react/issues/380

If the number of columns forced the console to be wider than the viewport, then the parent container was not sizing correctly. Added a min-width style that makes the parent container as wide as the console. Also saw that there were issues with resizing, added an option to auto-size using the autoFit property.

Storybook:
https://jschuler.github.io/patternfly-react/
Navigate to react-console -> SerialConsole

### CODE REVIEW

**Comment by jschuler** [comment]
@mareklibra Can't find you in list of reviewers

**Comment by mareklibra** [comment]
@jschuler , thanks for the fix and Cc me here.
LGTM

**Comment by mareklibra** [comment]
May I be added among reviewers? Like for similar cases.

File: packages/console/src/SerialConsole/SerialConsole.js [hunk 1]
```diff
@@ -110,7 +112,7 @@ class SerialConsole extends React.Component {
     }
 
     return (
-      <div className={topClassName} id={id}>
+      <div className={`serial-console-pf ${topClassName}`} id={id}>
```
Comment (jeff-phillips-18) [inline packages/console/src/SerialConsole/SerialConsole.js]: Better to use `ClassNames` to construct the className

File: packages/console/src/SerialConsole/__snapshots__/SerialConsole.test.js.snap [hunk 2]
```diff
@@ -33,7 +33,7 @@ exports[`Render SerialConsole in the CONNECTED state 1`] = `
 
 exports[`SerialConsole in the CONNECTED state 1`] = `
 <div
-  className=""
+  className="serial-console-pf "
```
Comment (jeff-phillips-18) [inline packages/console/src/SerialConsole/__snapshots__/SerialConsole.test.js.snap]: can we add a tests for adding the optional topClassName and autoFit parameters?

File: packages/console/src/SerialConsole/SerialConsole.js (hunk 1 above)
Comment (jschuler) [reply packages/console/src/SerialConsole/SerialConsole.js]: Thanks will switch to using ClassNames

File: packages/console/src/SerialConsole/__snapshots__/SerialConsole.test.js.snap (hunk 2 above)
Comment (jschuler) [reply packages/console/src/SerialConsole/__snapshots__/SerialConsole.test.js.snap]: Will add tests

**Comment by jeff-phillips-18** [comment]
@jschuler Could you use `yarn commit` so that the commit message has the correct format for the monorepo release?

**Comment by jschuler** [comment]
@jeff-phillips-18 removed a few lines of code that were not needed and updated the commit message

**Comment by jeff-phillips-18** [comment]
@mareklibra Would you like to review?

**Pull request merged**